src/EnergyPlus/DaylightingManager.cc

```diff
@@ -173,27 +172,22 @@ void DayltgAveInteriorReflectance(EnergyPlusData &state, int const enclNum) // E
     // REFERENCES:
     // Based on DOE-2.1E subroutine DAVREF
 
-    SurfaceClass IType; // Surface type/class
     Real64 AREA;        // Inside surface area (m2)
-    Real64 AInsTot;     // Total inside surface area of an enclosure (m2)
-    Real64 ARHTOT;      // Sum over surfaces of AREA*(inside visible reflectance) (m2)
-    int ITILT;          // Surface tilt category (1 = floor, 2 = wall, 3 = ceiling)
-    Real64 ATWL;        // Opaque surface area (m2)
-    Real64 ARHTWL;      // ATWL times inside visible reflectance of surface (m2)
     Real64 ETA;         // Ratio of floor-to-window-center height and average floor-to-ceiling height
 
     // Total inside surface area, including windows
-    AInsTot = 0.0;
+    Real64 AInsTot = 0.0;
     // Sum of products of inside surface area * vis reflectance
-    ARHTOT = 0.0;
+    Real64 ARHTOT = 0.0;
+    
     // Area sum and area * reflectance sum for different orientations
-    state.dataDaylightingManager->AR = 0.0;
-    state.dataDaylightingManager->ARH = 0.0;
+    std::array<Real64, (int)FWC::Num> AR = {0.0, 0.0, 0.0};
```

Comment:
These are local, pulled them out of `state.dataDaylightingManager`.

Reply:
Wonderful!!